.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "2.7"
+install:
+  - "pip install coveralls coverage --use-wheel"
+  - "pip install -e . --use-wheel"
```

Comment:
`--use-wheel` is now the default, just put `pip install --upgrade pip` as the first install to make sure you always have the latest.